client/landing/gutenboarding/available-designs.ts

```diff
@@ -56,7 +56,7 @@ export function prefetchDesignThumbs() {
 }
 
 export function getAvailableDesigns(
-	includeEdgeDesigns: boolean = isEnabled( 'gutenboarding/edge-templates' ),
+	includeEdgeDesigns: boolean = isEnabled( 'gutenboarding/alpha-templates' ),
```

Comment:
```suggestion
	includeAlphaDesigns: boolean = isEnabled( 'gutenboarding/alpha-templates' ),
```

Since we're changing the flag name, this might make sense too :)

Reply:
Just a test to see if you were paying attention :trollface: